sources/platform/actors/publishing/monetize.mdx

```diff
@@ -138,6 +122,42 @@ You can also choose not to use it, but then you must handle API integration and
+
+### Computing your costs for PPE and PPR Actors
+For both PPE and PPR Actors, profit is computed using the formula `(0.8 * revenue) - costs`. In this section, we'll explain how the `costs` component is calculated.
+
+When paying users run your Actor, it generates platform usage in the form of compute units, data traffic, API operations etc. This usage determines the `costs` in the profit formula above. Note that usage by free users is covered by Apify and doesn't affect your costs. To calculate costs in dollars, multiply the unit cost of each service by the quantity consumed. For example, if a run uses 10 compute units (CUs) at $0.5/CU, your cost would be $5.
+
+As mentioned in the [How to attract larger customers of PPE and PPR Actors](#how-to-attract-larger-customers-of-ppe-and-ppr-actors) section, the user's discount tier determines the unit costs applied to their runs if the Actor has tiered discounts set up. Higher tiers have lower unit costs, allowing you to offer lower prices to high-tier customers while maintaining healthy profit margins.
+
+The table below summarizes the unit prices for cost computation across different discount tiers.
+
+| Service                         | Price is per | `FREE`  | `BRONZE` | `SILVER` | `GOLD`  | `PLATINUM` |
+|:-------------------------------|:--------------|--------:|---------:|---------:|--------:|-----------:|
+| Compute unit                   | CU            | $0.4    | $0.4     | $0.3     | $0.25   | $0.1125    |
+| Residential proxies            | GB            | $8      | $8       | $7.5     | $7      | $4         |
+| SERPs proxy                    | 1,000 SERPs   | $2.5    | $2.5     | $2       | $1.7    | $1.25      |
+| Data transfer - external       | GB            | $0.2    | $0.2     | $0.19    | $0.18   | $0.18      |
+| Data transfer - internal       | GB            | $0.05   | $0.05    | $0.045   | $0.04   | $0.04      |
+| Dataset - reads               | 1,000 reads   | $0.0004 | $0.0004  | $0.00036 | $0.00032| $0.00032   |
+| Dataset - writes              | 1,000 writes  | $0.005  | $0.005   | $0.0045  | $0.004  | $0.004     |
+| Key-value store - reads       | 1,000 reads   | $0.005  | $0.005   | $0.0045  | $0.004  | $0.004     |
+| Key-value store - writes      | 1,000 writes  | $0.05   | $0.05    | $0.045   | $0.04   | $0.04      |
+| Key-value store - lists       | 1,000 lists   | $0.05   | $0.05    | $0.045   | $0.04   | $0.04      |
+| Request queue - reads         | 1,000 reads   | $0.004  | $0.004   | $0.0036  | $0.0032 | $0.0032    |
+| Request queue - writes        | 1,000 writes  | $0.02   | $0.02    | $0.018   | $0.016  | $0.016     |
+
+If you decide not to offer tiered discounts on your Actor, the unit prices for FREE tier apply.
+
```

Comment:
I think it would also make sense to add a section on using high Free pricing to limit usage for free users. It's one of the hidden benefits of the tiered pricing as well, that you can now customize Free pricing individually. Let me know if I should elaborate more.

Reply:
I actually considered whether we should explicitly advise this :D. At the end, I decided that the developers will do that regardless, even without our advice. But if you think it's a good idea, I'll add it there explicitly.

Reply:
Yeah, I think it will end up becoming the "best" practice even if we don't document it, so I'd get ahead of it and document it. Perhaps add something like:

While we don't recommend setting high prices for Free users by default, because it limits the users' ability to evaluate your Actor, we understand that sometimes it's necessary to protect you from fraud or excessive use of your internal APIs. For situations like this we recommend ....